drafts/2020-05-05-this-week-in-rust.md

```diff
@@ -18,6 +18,23 @@ If you find any errors in this week's issue, [please submit a PR](https://github
 
 * [Multi-threaded HTTP/WebSocket server in Rust](https://sergey-melnychuk.github.io/2020/04/27/multi-threaded-http-websocket-server-in-rust/).
 * [Cheating Rank-n with Traits](https://leshow.github.io/post/cheat_rank_n/)
+* [Understanding Memory and Thread Safety Practices
+and Issues in Real-World Rust Programs](https://cseweb.ucsd.edu/~yiying/RustStudy-PLDI20.pdf)
+* [A beginners guide to gRPC with Rust](https://dev.to/anshulgoyal15/a-beginners-guide-to-grpc-with-rust-3c7o)
+* [Setup Anaconda, Jupyter, and Rust](https://shahinrostami.com/posts/programming/rust-notebooks/setup-anaconda-jupyter-and-rust/)
+* [Teleforking a process onto a different computer!](https://thume.ca/2020/04/18/telefork-forking-a-process-onto-a-different-computer/)
+* [Rust at FullStory, Part 2: A Look Inside Our Mobile SDK](https://bionic.fullstory.com/rust-at-fullstory-part-2/)
+* [Stubbing out WASI manually in Rust](http://www.jakubkonka.com/2020/04/28/rust-wasi-from-scratch.html)
+* [Recon the Hydra Cave: a development note of Rust on GCC](https://nalaginrut.com/archives/2020/04/28/recon%20the%20hydra%20cave%3a%20a%20development%20note%20of%20rust%20on%20gcc%20%28part%201%29)
```

Comment:
This one feels much more like an opinion piece (with a lot of ranting) - I'd prefer not to include this one in the newsletter.

Reply:
Yes it is a recurring pattern now where some good how-to articles have a rant in the first half followed by interesting implementation details in the second. We observed the same with the web services post last week. Can we figure out how we can highlight the ideas presented in these rant-y posts?

Reply:
I'll give that some thought and write up some guidelines. To me, even if there is good technical content later in the post, starting off the post with a rant (especially ones related to community projects and teams) does disqualify it. If an author wants their technical content to be featured, it needs to not be accompanied by a rant. If they wish, they could create two posts - one with the ranty part and one with the technical how to part - and I'd be happy to feature the second one.

Reply:
It would be difficult to reach out to individual author's to maintain two versions of the same blog. Opinionated posts/articles could feature as an Alt-Rust section which could expose readers to explore different ideas and experiment with them. Please give it a thought.